terraform/cluster/mod/k8s/repo.sh

```diff
@@ -63,6 +63,8 @@ sudo sysctl --system
 # Install CRI-O
 if [ "$RUNTIME" = "crio" ]; then
   sudo apt install -y cri-o cri-o-runc
+  sudo rm /etc/cni/net.d/100-crio-bridge.conflist
```

Comment:
AFAIK, this file usually has the network cidr. Will all pods work if we remove these ? 
if so, Can I please look at the test and system/screenshot of "kubectl get pods -A" 
This will be helpful in mayastor-terraform-playground as well as I was facing issues with networking when using crio specially with hcloud csi pods